### DISCUSSION TITLE
Airports - how to deal with closed ones

### DISCUSSION THREAD

**Discussion by mxmtsk** [squirephp/squire#51] [Q&A] [answered]
I've noticed that the airport aata contains closed airports for example Berlin Tegel Airport which was closed about two years ago but also for example Berlin Tempelhof Airport which is closed for 14 years now.

I would argue that having a list of airports in operation is what most people will use it for, so my questions is if it's intended by the author of the package to contain closed airports and if so, if you think it would be a good idea to add a boolean column that indicates whether the airport is in operation?

**Comment by danharrin** [comment, marked as answer]
Yeah I would definitely add an `is_operative` column, but where would we find this data?

Ideally, we would also add a global scope to filter out non-operative airports by default, without breaking existing relationships.

**Reply by mxmtsk** [reply]
@danharrin Cool! I quickly searched and this could be a good base. The `type` column is `closed` if the airport is non-operative. The data is public domain as well. 

https://ourairports.com/data/

**Reply by danharrin** [reply]
Great, are you gonna submit a PR for the dataset then?

**Reply by mxmtsk** [reply]
@danharrin Yes, will take a while but I'll work on it, since I need that myself
